app/templates/administrative-units/administrative-unit/ministers/new.hbs

```diff
@@ -5,6 +5,26 @@
   <PageHeader class="au-u-margin au-u-margin-bottom-large">
     <:title>Voeg bedienaar toe</:title>
     <:subtitle>{{@model.administrativeUnit.name}}</:subtitle>
+    <:action>
+      {{! TODO: add "Voorwaarde" section once file uploads are supported }}
+      <AuButtonGroup class="au-c-button-group--align-right">
+        <AuButton
+          @icon="add"
+          @iconAlignment="left"
+          @loading={{this.createMinisterPositionTask.isRunning}}
+          @disabled={{this.createMinisterPositionTask.isRunning}}
+          type="submit"
+        >
+          Voeg toe
+        </AuButton>
+        <AuLink
+          @route="administrative-units.administrative-unit.ministers"
+          @skin="button-secondary"
+        >
+          Annuleer
+        </AuLink>
+      </AuButtonGroup>
+    </:action>
   </PageHeader>
 
   {{#if this.isSelectingTargetPerson}}
```

Comment:
This check also needs to be added before showing the buttons. At the moment they are visible before a person is selected. The same check is needed on the mandatory creation page.